### PR TITLE
Ensure signing/packaging works as expected

### PR DESCRIPTION
**Pull request by gcampbell-msft**
See title.

### CODE REVIEW

**Review by kuchungmsft — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -3677,7 +3677,6 @@
     "@test": "out/test/"
   },
   "scripts": {
-    "vscode:prepublish": "yarn run compile-production",
```
Comment (benmcmorran) [inline package.json:3680]: Why is this removed? Will that break external contributors trying to package the VSIX themselves?

**Review by benmcmorran — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (gcampbell-msft) [reply package.json:3680]: No, it just means that they'll need to do the same as our pipeline, `yarn run compile-production`, and then `yarn package` or the `vsce package` command

We have to remove this because in our pipeline, when we run `vsce package` it will do the `vscode:prepublish` command, which rebuilds and regenerates loc files, etc. This means that in our pipelines, the files that we're signed are getting corrupted when we try to package
